modules/default/newsfeed/newsfeed.js

```diff
@@ -5,7 +5,9 @@ Module.register("newsfeed", {
 			{
 				title: "New York Times",
 				url: "https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml",
-				encoding: "UTF-8" //ISO-8859-1
+				encoding: "UTF-8", //ISO-8859-1
+				ignoreOldItems: false, 
```

Comment:
Rather than having ignorOldItems and ignorOlderThan be global parameters, I added them inside the feed array and gave then default values of false and 1day.